pkg/roachpb/api.proto

```diff
@@ -698,6 +698,43 @@ message DeprecatedVerifyChecksumResponse {
   optional ResponseHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
 }
 
+enum ExportStorageProvider {
+  Unknown = 0;
+  LocalFile = 1;
+  //Http = 2;
```

Comment:
I think all mentions of HTTP should be removed and the protobuf numbers decreased to compensate. This is because we might determine in the future that HTTP doesn't make sense as a backup/restore location because it doesn't have good support for directory lists or writes. I think holding a slot for it will be an oddity in the code that confuses more than helps.